### ISSUE TITLE
Documentation: provide a documentation about how this module work and what data is sent to GA

### ISSUE THREAD

**Issue by matks** [PrestaShop/PrestaShop#26102]
#### Is your feature request related to a problem?
In order to be able to provide accurate and extended configuration capabilities / data synchronization between PS and GA, it would be useful to provide a documentation about how this module work and what data is sent to GA.

#### Describe the solution you'd like
The doc should explain the inner working of the module.
The doc should explain what data are being sent from PS to GA, and where.

Once the list is complete, it'll provide an accurate overview of what data is missing / can be expanded (see https://github.com/PrestaShop/ps_googleanalytics/issues/57) and maybe open the path for advanced configuration settings (being able, from BO, to enable/disable what is being sent)

**Comment by Hlavtox** [comment]
No longer relevant, GA is implemented following https://developers.google.com/analytics/devguides/collection/ga4/ecommerce?client_type=gtag
